packages/block-editor/src/components/block-button/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RichText from '../rich-text';
+
+export default class BlockButton extends Component {
+	render() {
+		const newProps = { ...this.props };
+		newProps.className += ' wp-element-button';
+
+		// If children are set then we only need a button, not RichText.
+		if ( newProps.children ) {
+			return <button { ...newProps } />;
+		}
+
+		if ( this.props.frontEnd ) {
```

Comment:
This should probably be a separate component `ButtonElement.Content` (like the other similar components we have: RichText, InnerBlocks)

Reply:
I had a go at doing this, but I'm not sure if there's a better way.